docs/data/date-pickers/localization/localization.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Pickers - Localization
+---
+
+# Pickers - Localization
+
+<p class="description">Pickers allow to support users from different locales, with formatting, RTL, and localized strings.</p>
```

Comment:
```suggestion
<p class="description">Date and time pickers allow to support users from different locales, with formatting, RTL, and localized strings.</p>
```

The official naming of the project is "Date and time pickers"
I think we can shortcut with "pickers" when inside a page that already clearly scoped the topic.
But for a title / description we should probably be explicit

____

Same for the title I think

On the Getting Start page we have "Date / Time pickers" which is not great I think (https://mui.com/x/react-date-pickers/getting-started/)

@samuelsycamore I you have a preference